protocol-impl/src/main/java/io/zeebe/protocol/impl/encoding/MsgPackConverter.java

```diff
@@ -178,4 +178,16 @@ public static String convertJsonSerializableObjectToJson(JsonSerializable record
       throw new RuntimeException(e);
     }
   }
+
+  ////////////////////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////// MAP TO JSON //////////////////////////////////////////////
+  ////////////////////////////////////////////////////////////////////////////////////////////////
+
+  public static String convertToJson(Map<String, Object> map) {
```

Comment:
Why we need this?

Reply:
We don't, I forgot to remove that dead code. Fixed it now